testkit/src/builder.rs

```diff
@@ -111,6 +111,8 @@ pub struct TestKitBuilder {
     validator_count: Option<u16>,
     service_instances: Vec<InstanceCollection>,
     logger: bool,
+    runtime_factories: Vec<Box<dyn RuntimeFactory>>,
+    instances: Vec<InstanceConfig>,
```

Comment:
What's the difference between service_instances (also holding InstanceConfig as a field) and instances? Shall `instances` include all the things stored inside `service_instances`?

Reply:
Even more, I suppose that the Rust instances should also use `RuntimeFactory` in common way like other Runtimes.

Reply:
In short - because of support of testkit restart. As soon as we remove StoppedTestkit - it will simplify the whole schema.